apis/Google.Cloud.Spanner.Data/docs/index.md

```diff
@@ -254,3 +254,34 @@ If you want to diagnose session management issues,
 the statistics for the session pool associated with the connection:
 
 {{sample:SpannerConnection.GetSessionPoolSegmentStatistics}}
+
+## RpcException from SpannerException
+
+The `RpcException` property of `SpannerException` returns the underlying
+`RpcException` which caused the `SpannerException`. The returned 
+value may be null if the `SpannerException` was not caused by an `RpcException`.
+
+`ErrorInfo` in [RpcException](../../../docs/devsite-help/troubleshooting.md#how-can-i-get-errorinfo-from-rpcexception) can 
+provide actionable error deatils. 
```

Comment:
```suggestion
This can in turn be used to [obtain an ErrorInfo](../../../docs/devsite-help/troubleshooting.md#how-can-i-get-errorinfo-from-rpcexception) which can provide actionable error details. 
```